contrib/dash/actions/script-osx.sh

```
@@ -8,13 +8,10 @@ export PATH=$PATH:$PY37BINDIR
 
-if [[ -n $GITHUB_REF ]]; then
-    PIP_CMD="sudo python3 -m pip"
-else
-    python3 -m virtualenv env
-    source env/bin/activate
-    PIP_CMD="pip"
-fi
+python3 -m venv env
+source env/bin/activate
+python3 -m pip install --quiet --upgrade pip
++PIP_CMD="python3 -m pip"
```

Shellcheck (0.10.0)

[error] This is interpreted as a command name containing '='. Bad assignment or comparison?

(SC2276)